deploy/kubernetes/wait-for-driver.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# This script waits for the deployments from ./cluster_setup.sh to be ready.
+
+controller_statefulset=gcp-filestore-csi-controller
```

Comment:
should we use deployment instead of statefulset?

Reply:
Done